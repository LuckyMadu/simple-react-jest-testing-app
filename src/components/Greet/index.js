import React from "react";
import "./style.css";

const Greet = () => {
  return (
    <div id="greet">
      <h1>Greetings!!!</h1>
    </div>
  );
};

export default Greet;
