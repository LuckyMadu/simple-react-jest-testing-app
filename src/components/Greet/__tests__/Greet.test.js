import React from "react";
import renderer from "react-test-renderer";
import Greet from "../index";

describe("Greet ui tests", () => {
  test("ui rendered correctly", () => {
    const component = renderer.create(<Greet />).toJSON;
    expect(component).toMatchSnapshot();
  });
});
