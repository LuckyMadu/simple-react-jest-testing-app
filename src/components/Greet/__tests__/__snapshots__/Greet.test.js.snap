// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greet ui tests ui rendered correctly 1`] = `[Function]`;
